src/test/java/redis/clients/jedis/tests/commands/BinaryValuesCommandsTest.java

```diff
@@ -116,6 +116,16 @@ public void setAndExpire() {
     assertTrue(ttl > 0 && ttl <= expireSeconds);
   }
 
+  @Test
+  public void setAndKeepttl() {
+    String status = jedis.set(bfoo, binaryValue, setParams().nx().ex(expireSeconds));
+    assertTrue(Keyword.OK.name().equalsIgnoreCase(status));
+    status = jedis.set(bfoo, binaryValue, setParams().keepttl());
+    assertTrue(Keyword.OK.name().equalsIgnoreCase(status));
+    long ttl = jedis.ttl(bfoo);
+    assertTrue(0 < ttl && ttl <= expireSeconds);
+  }
```

Comment:
`set` again, this time without any modifier and check that ttl is gone.

Reply:
Thanks. fixed at lastest commit.